Doc/c-api/tuple.rst

```diff
@@ -35,6 +35,10 @@ Tuple Objects
 
    Return a new tuple object of size *len*, or *NULL* on failure.
 
+   .. versionchanged:: 3.7
+
+      ``PyTuple_New(0)`` now does not raise exceptions.
```

Comment:
Yes, because it always succeeds. It now can be used at very early stage of initialization and at very late stage of shutdown. Could you suggest better wording?